server/src/test/java/org/hyperskill/community/flashcards/integration/ExampleDataInitializerIT.java

```diff
@@ -0,0 +1,29 @@
+package org.hyperskill.community.flashcards.integration;
+
+import org.hyperskill.community.flashcards.TestMongoConfiguration;
```

Comment:
I like this way of separating the IT tests into t a package - but leave them under test-tree !
Will adapt this in my projects in the future

Reply:
Sorry for being dumb, but could you explain what you mean? :grinning:

Reply:
we clarified in chat :-)